pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -1272,6 +1273,44 @@ private static void TeamCityFinishTestSuite(CommandLineArgs commandLineArgs)
                 unfilteredTestList.RemoveAll(test => test.IsPerfTest);
             }
 
+            if (asNightly)

```

Comment:
Can this please be put in a function? RunTestPasses() is already a monster of a function and this won't make it any easier to read the main thrust of its purpose.